src/data/html-quiz.ts

```diff
@@ -1627,5 +1627,16 @@ const htmlQuiz = [
       "The <i> tag is used to make text italic without adding extra emphasis. It is commonly used for text that is set off from normal prose, like foreign words or thoughts. ",
     Link: "https://devdocs.io/html/element/i",
   },
+  {
+    Question:
+      "What is the use of iframes?",
+    Answer: "embed another document within the current HTML document ",
+    Distractor1: "used for creating a hyperlink",
+    Distractor2: "used to italicize text",
+    Distractor3: "a tag used for preformatted sections of text",
+    Explanation:
+      "You can embed another document within the current HTML document using <iframe> tag. It provides a nested browsing context without using another document by just passing the content to the <iframe> via the srcdoc attribute.",
+    Link: "https://www.tutorialride.com/html-interview/html-interview-questions-part-5.htm",
```

Comment:
Please only use freeCodeCampe resources, MDN or wikipedia articles 

```suggestion
    Link: "https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe",
```